adtl/transformations.py

```diff
@@ -0,0 +1,19 @@
+""" Functions which can be applied to source fields, allowing extensibility """
+
+from datetime import datetime
+import pint
+
+
+def isNotNull(value):
+    True if value != None and value != "" else False
+
+
+def yearsElapsed(birthdate, currentdate):
+    if birthdate == (None or "") or currentdate == (None or ""):
```

Comment:
`birthdate in`, `currentdate in`, the right hand side of equality will resolve to "" so None is not being compared.